environs/config/config.go

```diff
@@ -1735,6 +1735,13 @@ func (c *Config) ValidateUnknownAttrs(extrafields schema.Fields, defaults schema
 				continue
 			case float64:
 				continue
+			case []interface{}:
```

Comment:
Would like to see a test for this.

Reply:
Done https://github.com/juju/juju/pull/12183/commits/1f1b2108f89509d09e49cafbce7c0aeaf530ad60